test-requirements.txt

```diff
@@ -1,3 +1,3 @@
 PyVirtualDisplay==0.1.5
-selenium==2.49.0
+selenium==2.49.2
```

Comment:
Odd - [they passed on CircleCI](https://circleci.com/gh/OpenTreeMap/otm-core/837), I didn't run them locally.

I checked and CircleCI _did_ install 2.49.2 of selenium....